src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -35,6 +35,9 @@
 		<None Update="Manifest\exampleManifest.json">
 			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
 		</None>
+		<None Update="Preload\fdc3-iife-bundle.js">
```

Comment:
`<Content Include="..."`
See https://learn.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2022

Reply:
Shouldn't this be an embedded resource instead?

Reply:
You're right, we already discussed and agreed to embed it.

Reply:
It's configured to be an Embedded Resource now.